src/node-http2-core-inl.h

```diff
@@ -195,6 +194,13 @@ inline bool nghttp2_session_find_stream(
   }
 }
 
+inline void nghttp2_set_callbacks_allocate_send_buf(
+    node_nghttp2_session_callbacks* callbacks,
+    nghttp2_allocate_send_buf_cb cb) {
+  assert(callbacks != nullptr);
+  callbacks->allocate_send_buf = cb;
+}
```

Comment:
started this by following the pattern nghttp uses but I've been wanting to refactor it to simplify.